Movio/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/colorPrimaryAccent"
```

Comment:
pozadie ma mat farbu accent color definovanu prave zvolenou temou. 
tu je priamo nastavena konkretna farba definovana v colors.

ci to spravne funguje, sa da overit castou nasledujucou za bodom 7 v ulohe.

Reply:
Ahoj opravil som to + to z prvého úkolu z ISu.

Ešte som sa chcel opýtať, dal som naopak poradie itemov v packagoch mám to prerobiť alebo to už nevadí?